tests/qiskit/test_transpiler.py

```diff
@@ -226,6 +227,20 @@ def test_coloring_sample_decode():
     assert sampleset[0].var_values["x"].values == {(0, 0): 1, (1, 0): 1, (2, 0): 1, (3, 0): 1}
     assert sampleset[0].num_occurrences == 1024
 
+def test_parametric_exp_gate(transpiler: QiskitTranspiler):
```

Comment:
You should also test multi term hamiltonian such as $X_0Y_1 - Z_0 X_1$

Reply:
should I add the second term to the existing Hamiltonian` X_0 Z_1`? Or make another new Hamiltonian such as `hamiltonian_2`?